action.yml

```diff
@@ -19,9 +23,25 @@ runs:
       shell: pwsh 
       run: |
         # Windows SQL install via Chocolatey
+        $template = @'
+        {[string]Package*:sql-server-express}|{[int]Major:2022}.{[int]Minor:16}.{[int]Patch:0}.{[int]Build:1000}
+        {[string]Package*:sql-server-express}|{[int]Major:2022}.{[int]Minor:16}.{[int]Patch:0}.{[int]Build:0}
+        {[string]Package*:sql-server-express}|{[int]Major:14}.{[int]Minor:1801}.{[int]Patch:3958}.{[int]Build:1}
+        '@
+
+        # Limited output should be sql-server-express|versionstring
+        $availableVersions = choco search sql-server-express --exact --all-versions --limit-output | Where-Object { $_ -like "*${{ inputs.sqlserver-version }}**" }
+        # IncludeExtent will feed the input into an ExtendText property. Using the template examples helps to order by Major which is all we are interested in
+        $sortedVersions = $availableVersions | ConvertFrom-String -TemplateContent $template -IncludeExtent | Sort-Object -Property Major -Descending
+        $newestVersion = $sortedVersions[0].ExtentText.Split("|")[1]
 
-        echo "Installing SQL Server Express with Chocolatey..."
-        choco install sql-server-express --no-progress
+        if ($newestVersion) {
+            Write-Host "Installing SQL Server Express version $newestVersion with Chocolatey..."
+            choco install sql-server-express --version $newestVersion -y --no-progress
+        } else {
+            Write-Host "No versions found matching the prefix '$VersionPrefix'. Installing latest with Chocolatey..."
```

Comment:
Is `$VersionPrefix` a variable because this is the only place it is mentioned? Should this be `${{ inputs.sqlserver-version }}` instead?

Reply:
Done